### PR TITLE
Jurisdiction Admin separate routes

### PR DESCRIPTION
**Pull request by jonahkagan**
**Description**

Move jurisdiction admin routes to `/election/:electionId/jurisdiction/:jurisdictionId` so that we can support cases where a JA is part of multiple audits.

Also clean up some other routing stuff and update a few tests that were failing.

### CODE REVIEW

File: arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const QRs: React.FC<{ electionId: string; boardIds: string[] }> = ({
       {boardIds.map(id => (
         <span key={id} id={`qr-${id}`}>
           <QRCode
-            value={`${window.location.origin}/election/${electionId}/board/${id}`}
+            value={`${window.location.origin}/election/${electionId}/audit-board/${id}`}
```
Comment (benadida) [inline arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx:61]: Hmmm, maybe this slipped in in a prior change, but isn't this QR code supposed to link to `/auditboard/{passphrase}` ?

File: arlo-client/src/components/CreateAudit.tsx [hunk 2]
```diff
@@ -79,6 +79,11 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       toast.error(err.message)
     }
   }
+
+  if (isAuthenticated === null) return null // Still loading
```
Comment (benadida) [inline arlo-client/src/components/CreateAudit.tsx:83]: I know this `isAuthenticated` variable isn't introduced in this PR, but it's a bit fragile to treat a boolean as a trinary value where `false` and `null` aren't the same thing. We should probably fix this at some point.

File: arlo-client/src/components/CreateAudit.tsx [hunk 3]
```diff
@@ -79,6 +79,11 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
...
+  if (meta && meta.type === 'audit_board') {
+    return <div>404 Not Found</div>
```
Comment (benadida) [inline arlo-client/src/components/CreateAudit.tsx:85]: is this a fake 404? :)

File: arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx (hunk 1 above)
Comment (jonahkagan) [reply arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx:61]: dunno, they end up at the same place though. should i change it?

File: arlo-client/src/components/CreateAudit.tsx (hunk 2 above)
Comment (jonahkagan) [reply arlo-client/src/components/CreateAudit.tsx:83]: good call. i actually think the variable is unnecessary. i explored refactoring a bit to having a user type that included anonymous users, and then just returning the user from this hook (`null` if still loading). but i agree it's not a priority

File: arlo-client/src/components/CreateAudit.tsx (hunk 3 above)
Comment (jonahkagan) [reply arlo-client/src/components/CreateAudit.tsx:85]: yeah, before this view (which is at the root path) was crashing if you were logged in as an audit board, so i just wanted to put in something slightly better

File: arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx (hunk 1 above)
Comment (benadida) [reply arlo-client/src/components/Audit/CalculateRiskMeasurement.tsx:61]: so the URL you have should *not* authenticate the audit board user, I think? I wouldn't want someone seeing a URL on a screen be enough to log in as the audit board. The passphrase URL is the one that should set the cookie.

**Pull request merged**